src/components/my-journey/ContestDetail.tsx

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
`Line 19:` "It is better to use isModalVisible as the variable name for a boolean

Reply:
i agree

Reply:
I feel that it would be better if the skeleton code were extracted as a component. It is kind of confusing to see div tags without any children or text inside them.